network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -91,10 +94,20 @@ class BlogIndexPage(IndexPage):
     with additional logic to explore topics.
     """
 
+    related_topics = ParentalManyToManyField(
```

Comment:
**Nitpick** I like to define the config options (like `subpage_types` and `base_form_class`) before the fields, but that's only a matter of personal taste (so consider this totally optional).

Reply:
I agree that this sounds like a good idea! I have added this to the latest commit